about.html

```diff
@@ -0,0 +1,155 @@
+                        those needs. Software itself can become a catalyst for change: business change, cultural change
+                        – even world change. Rather than a liability or sunk cost, applications can become a source of
+                        innovation and competitive advantage.
+                    </p>
+                </div>
+                <div id="conference-contact" class="p-content-desc">
+                    <p>Plase contact us per Email for any futher questions about New Technologies Conference</p>
+                    <p class="cc-email"> <a href="mailto:contact@ntc.com">contact@ntc.com</a></p>
+                </div>
+            </div>
+        </div>
+        <div id="header-img-container" class="full-size">
+
+            <div id="header-img" class="full-size">
+
+            </div>
+        </div>
+        <div id="main-header-overlay" class="full-size">
+
+        </div>
+    </header>
+    <section id="about-logo" class=" full-size">
+        <div class="content section-container">
+            <div class="section-head">
+                <h2>New Technologies Conference Logo</h2>
+            </div>
+            <div class="section-content">
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Beatae sint modi praesentium, pariatur
+                    debitis amet ut quia voluptas, deleniti adipisci nemo autem dolores obcaecati doloribus. Vitae qui
+                    quis numquam ab.
+                    Iste eos obcaecati tempora aut, itaque beatae assumenda voluptatem animi tempore mollitia
+                    consequatur, sit nihil temporibus error voluptatibus provident dicta dolorum. Labore accusamus
+                    possimus nemo. Enim fugiat voluptate soluta corrupti?</p>
+                <div class="main-text">
+
+                    <div class="logo">New Technologies <br> Conference</div>
+                </div>
+            </div>
+
+        </div>
+    </section>
+    <section id="last-conf" class="full-size">
+        <div class="content section-container">
+            <div class="section-head">
+                <h2>See the past Conferences</h2>
+            </div>
+            <div class="section-content">
+                <p>
+                    Take a look at last two New Technolgies Conferences
+                </p>
+                <div class="lc-img-container">
+                    <div class="lcimg-item">
+                        <img src="images/4e4df3bbabf7db484421.jpg" alt="Image from conference 2020">
+                        <div class="lcm-overlay full-size">
+                            <h4>2020</h4>
+                            <p>NTC in Amsterdam</p>
+                        </div>
+                    </div>
+                    <div class="lcimg-item">
+                        <img src="images/972fc8e9f59af47cf5e0.jpg" alt="Image from conference 2021">
```

Comment:
- [ ] In the mobile screens the two images of the conferences should be aligned vertically as the mobile design so I encourage you to do that